dashboard/app/models/ai_tutor_interaction.rb

```diff
@@ -0,0 +1,33 @@
+# == Schema Information
+#
+# Table name: ai_tutor_interactions
+#
+#  id                 :bigint           not null, primary key
+#  user_id            :integer          not null
+#  level_id           :integer
+#  script_id          :integer
+#  ai_model_version   :string(255)
+#  type               :string(255)
+#  project_id         :string(255)
+#  project_version_id :string(255)
+#  prompt             :text(65535)
+#  status             :string(255)
+#  ai_response        :text(65535)
+#  created_at         :datetime         not null
+#  updated_at         :datetime         not null
+#
+# Indexes
+#
+#  index_ai_tutor_interactions_on_level_id   (level_id)
+#  index_ai_tutor_interactions_on_script_id  (script_id)
+#  index_ai_tutor_interactions_on_user_id    (user_id)
+#  index_ati_user_level_script               (user_id,level_id,script_id)
+#
+class AiTutorInteraction < ApplicationRecord
+  self.inheritance_column = nil
```

Comment:
🤓 learning moment: when you have a column named `type` Rails automatically thinks you're invoking Single Table Inheritance and that the type column will define which subclass an object belongs to. In this line, I'm telling Rails that I'm not using inheritance. Tuck away for the future that something more specific like `tutor_type` would be a better choice.